lib/matplotlib/artist.py

```diff
@@ -297,13 +316,11 @@ def add_callback(self, func):
 
     def remove_callback(self, oid):
         """
-        Remove a callback based on its *id*.
-
-        .. seealso::
-
-            :meth:`add_callback`
-               For adding callbacks
+        Remove a callback based on its observer *id*.
```

Comment:
`oid`, not `id`

Reply:
Changed to "observer id" without markup. "observer *oid*" would be sort of double.